src/components/video-list/video.js

```diff
@@ -0,0 +1,143 @@
+import { useState } from "react";
+import { useAuth } from "../../contexts/auth-context";
+import { useData } from "../../contexts/data-context";
+import { useNavigate } from 'react-router-dom';
+import axios from "axios";
+
+// add to Liked videolist
+const addToLikedVideos = async (video, token) => {
+    return await axios.post("/api/user/likes",
+    {
+        video: video 
+    },
+    {
+        headers: {
+            authorization: token
+        }
+    });
+}
+
+// remove from liked videolist
+const removeFromLikedVideos = async (videoId, token) => {
+    return await axios.delete(`/api/user/likes/${videoId}`,
+    {
+        headers: {
+            authorization: token
+        }
+    });
+}
+
+// add to Liked videolist
+const addToWatchLaterVideos = async (video, token) => {
+    return await axios.post("/api/user/watchlater",
+    {
+        video: video 
+    },
+    {
+        headers: {
+            authorization: token
+        }
+    });
+}
+
+// remove from liked videolist
+const removeFromWatchLaterVideos = async (videoId, token) => {
```

Comment:
it's a good practice to store all your services in separate file so it becomes more legible

Reply:
Thanks for suggestion, I will change it in next PR which is related to watchlater